generator/CANlib_h.py

```diff
@@ -50,17 +50,26 @@ def write(can, output_path=can_lib_h_path):
             fw('typedef enum {\n')
 
             for msg in bus.frame:
-                fw(templ['enum'].format(coord(bus.name, msg.name), idx))
-                idx += 1
+                if isinstance(msg, ParseCAN.spec.bus.MultiplexedFrame):
+                    for frame in msg.frame:
+                        fw(templ['enum'].format(coord(bus.name, msg.name, frame.name), idx))
+                        idx += 1
```

Comment:
I don't think we need the idx. You can just print the members without `= idx`.

Reply:
This is because they don't start at 0, instead you have the can error msg and unknown msg

Reply:
You can have mixed assigned and not assigned i think

Reply:
That works okay for one bus, but I think the original intent was to avoid repeated values across different buses. This was the comment that went with it:
```python
# Begin enumerating from the universal forms onward
# Every time we assign to an enum, we'll increment beg
# so as to avoid return value equalities between different bus
```
That being said, I don't think this is actually necessary...so I will change it